content/information-distribution/article.md

```diff
@@ -32,9 +32,9 @@ module](https://matrix.org/docs/spec/client_server/r0.4.0.html#id112).
 
 ## Matrix event `network.informo.article`
 
-This message event is meant to be sent by a source Matrix user. If the article
-is signed, it **must** be wrapped into a [Signed Matrix
-event](/information-distribution/signature/#signed-matrix-event).
+This message event **must** be sent and signed by a source Matrix user. See
+[signed Matrix event](/information-distribution/signature/#signed-matrix-event)
```

Comment:
Sorry to be picky here, but I think the rendered version of this page might not make the face that we refer to another page obvious enough. I'd rephrase that into "See the [signed Matrix event] page for more information...".